src/qibolab/instruments/zhinst.py

```diff
@@ -808,8 +785,7 @@ def play_sweep_select_single(exp, qubit, pulse, section, parameters, partial_swe
             exp.play(
                 signal=f"{section}{qubit.name}",
                 pulse=pulse.zhpulse,
-                phase=pulse.zhsweeper,  # I believe this is the global phase sweep
-                # increment_oscillator_phase=pulse.zhsweeper, # I believe this is the relative phase sweep
```

Comment:
For this I could not reach a conclusion on which to use, as we were not sweeping phase anywhere

Reply:
Do you want me to keep a small comment, so that we can keep an eye on it in the future?